src/index.js

```diff
@@ -168,7 +168,7 @@ function renderSubtreeIntoContainer(parentComponent, vnode, container, callback)
 	let wrap = h(ContextProvider, { context: parentComponent.context }, vnode);
 	let c = render(wrap, container);
 	if (callback) callback(c);
-	return c;
+	return c._component || c.base;
```

Comment:
Strange - it seems like this should have already been taken care of by [L154](https://github.com/developit/preact-compat/pull/295/files#diff-1fdf421c05c1140f6d71444ea2b27638L154), shouldn't it?  I wonder why it wasn't working...